src/Geometry/Grids.jl

```diff
@@ -265,6 +265,14 @@ function compute_reference_grid(reffe::LagrangianRefFE, nelems::Integer)
   compute_reference_grid(p,nelems)
 end
 
+"""
+    compute_reference_grid(p::LagrangianRefFE, partition::NTuple{D,Integer})
+"""
+function compute_reference_grid(reffe::LagrangianRefFE{D}, partition::NTuple{D,Integer}) where D
```

Comment:
Is this method needed? If yes, where?

Reply:
I implemented this method to complete the set. We had 3 already implemented signatures: 
- `LagrangianFE`+`Int`
- `Polytope`+`Int`
- `Polytope`+`Tuple`
I just added the one that is missing. It is used when creating a `RefinementRule` starting from a `LagrangianFE`. I also have all 4 signatures for convenience: 
```
function RefinementRule(reffe::LagrangianRefFE{D},nrefs::Integer;kwargs...) where D
  partition = tfill(nrefs,Val{D}())
  return RefinementRule(get_polytope(reffe),partition;kwargs...)
end

function RefinementRule(reffe::LagrangianRefFE{D},partition::NTuple{D,Integer};kwargs...) where D
  return RefinementRule(get_polytope(reffe),partition;kwargs...)
end

function RefinementRule(poly::Polytope{D},nrefs::Integer;kwargs...) where D
  partition = tfill(nrefs,Val{D}())
  return RefinementRule(poly,partition;kwargs...)
end

function RefinementRule(poly::Polytope{D},partition::NTuple{D,Integer};kwargs...) where D
  ref_grid = UnstructuredGrid(compute_reference_grid(poly,partition))
  return RefinementRule(GenericRefinement(),poly,ref_grid;kwargs...)
end
```